### ISSUE TITLE
ardulink.org is down

### ISSUE THREAD

**Issue by jpink** [Ardulink/Ardulink-2#84]
The https://www.ardulink.org/ web page responds blank white page but http://arduinopclink.altervista.org (which I see on the source) seems to be working.

**Comment by LucianoZu** [comment]
Yes, I'd the same experience but just the first time. When I tried to see into administration pages the problem solved itself. Could you retry please?

**Comment by jpink** [comment]
Still the same issue.

**Comment by LucianoZu** [comment]
Hmmm, it seems a different behaviour between Firefox, Edge and Chrome. Just with Chrome I can't see the site. It's very strange. I'll try to ask for assistance to the host company.
